cmd/command.go

```diff
@@ -505,6 +505,18 @@ func hScan(cli *Client, args [][]byte) (interface{}, error) {
 	return cli.db.HScan(args[0], args[1], pattern, count)
 }
 
+func hIncrby(cli *Client, args [][]byte) (interface{}, error) {
```

Comment:
It is good to follow Go naming convention. 
```suggestion
func hIncrBy(cli *Client, args [][]byte) (interface{}, error) {
```